tiledb/sm/array/array_directory.cc

```diff
@@ -765,16 +802,23 @@ ArrayDirectory::compute_filtered_uris(
     if (is_vacuum_file(uri))
       continue;
 
```

Comment:
Nit: add a comment to describe each block of codes... in this case:
// Get the fragment version.